spec/design/skel/AARCH64/ArchInterrupt_H.thy

```diff
@@ -15,6 +15,10 @@ begin
 
 context Arch begin global_naming AARCH64_H
 
+(* Kernel_Config provides a generic numeral, Haskell expects type irq *)
+abbreviation (input) maxIRQ :: irq where
+  "maxIRQ == Kernel_Config.maxIRQ"
```

Comment:
Why is this abbreviation introduced in both `ArchInterrupt_H` and `Hardware_H`? Are the imports too complicated here?

Reply:
This looks like like a left-over to be honest, the one in `ArchInterrupt_H` can probably be removed. Will look into it in a separate PR.